.eslintrc.yml

```
@@ -1,30 +1,20 @@
 env:
   browser: true
   es6: true
-extends: 'eslint:recommended'
+extends:
+  - eslint:recommended
+  - prettier
 parserOptions:
   sourceType: module
+  ecmaVersion: 2017
```

Tu  w sumie powinno być `latest`. Daje sugestię i przeniosę później do generatorów
```suggestion
  ecmaVersion: latest
```

---

```suggestion
  ecmaVersion: 2022
```

---

Zgodnie z rozmową na Slack, dajmy tutaj jednak 2022 żeby zapobiec errorom przy automatycznej zmianie wersji na najnowszą
```suggestion
  ecmaVersion: 2022
```